packages/app-bridge-theme/package.json

```diff
@@ -0,0 +1,63 @@
+{
+    "name": "@frontify/app-bridge-theme",
+    "type": "module",
+    "version": "1.0.0-alpha.1",
+    "description": "Package to establish communication between Frontify and themes",
+    "author": "Frontify Developers <developers@frontify.com>",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/Frontify/brand-sdk",
+        "directory": "packages/app-bridge-theme"
+    },
+    "sideEffects": false,
+    "exports": {
+        ".": {
+            "types": "./dist/index.d.ts",
+            "require": "./dist/index.js",
+            "import": "./dist/index.js"
+        }
+    },
+    "main": "./dist/index.js",
+    "types": "./dist/index.d.ts",
+    "files": [
+        "dist",
+        "package.json.d.ts"
+    ],
+    "engines": {
+        "node": ">=16"
+    },
+    "scripts": {
+        "build": "ts-json-as-const ./package.json && vite build",
+        "dev": "vite build --watch",
+        "lint": "eslint .",
+        "lint:fix": "eslint --fix .",
+        "test": "vitest run",
+        "test:ui": "vitest --ui",
+        "test:watch": "vitest",
+        "test:coverage": "vitest run --coverage",
+        "typecheck": "tsc --noEmit"
+    },
+    "peerDependencies": {
+        "react": "^18",
+        "sinon": "^15 || ^16 || ^17"
+    },
+    "dependencies": {},
+    "devDependencies": {
+        "@frontify/eslint-config-react": "^0.17.6",
+        "@testing-library/react": "^14.3.0",
+        "@types/react": "^18.2.75",
+        "@types/react-dom": "^18.2.24",
+        "@vitest/coverage-v8": "^1.4.0",
+        "@vitest/ui": "^1.4.0",
+        "eslint": "^8.57.0",
+        "eslint-plugin-notice": "^0.9.10",
+        "prettier": "^3.2.5",
+        "react": "^18.2.0",
+        "ts-json-as-const": "^1.0.7",
+        "type-fest": "^4.15.0",
+        "typescript": "^5.4.4",
+        "vite": "^5.2.10",
+        "vite-plugin-dts": "^3.8.1",
+        "vitest": "^1.4.0"
```

Comment:
We currently don't have any tests or code to be tested -> Remove test-related dependencies